server/application/logs.go

```diff
@@ -39,6 +39,11 @@ func parseLogsStream(podName string, stream io.ReadCloser, ch chan logEntry) {
 		timeStampStr := parts[0]
 		logTime, err := time.Parse(time.RFC3339Nano, timeStampStr)
 		if err != nil {
+			if strings.HasPrefix(line, "unable to retrieve container logs for ") ||
+				strings.HasPrefix(line, "Unable to retrieve container logs for ") {
+				ch <- logEntry{line: line, podName: podName, timeStamp: time.Now()}
```

Comment:
Should err still be added here?

Reply:
No, any error in the log channel would stop all other streams immediately.
https://github.com/argoproj/argo-cd/blob/728b31e5e9fad96a4d381c7e6712b102d8f3fa5c/server/application/application.go#L1796-L1804